lms/templates/theme-variables.html

```diff
@@ -1,17 +1,42 @@
 ## mako
+<%! import urllib %>
 <%! from django.core.urlresolvers import reverse %>
 <%! from openedx.core.djangoapps.site_configuration.helpers import get_current_site_configuration, get_value %>
 <%! from openedx.core.djangoapps.site_configuration.models import SiteConfiguration %>
 <%! from django.contrib.staticfiles.templatetags.staticfiles import static %>
-<%! from datetime import datetime %>
+<%! from django.utils.encoding import force_text %>
+<%! from django.utils import translation %>
+<%! from django.utils.translation import ugettext, ugettext_lazy as _ %>
+<%! from django.utils.functional import Promise %>
+<%! from django.utils import six %>
+<%! from django.utils.functional import lazy %>
+<%! from datetime import date %>
```

Comment:
@bryanlandia I think this should be `<%! from datetime import datetime, date %>` We use `datetime.now()` to get current year https://github.com/appsembler/edx-theme-customers/blob/648708c85452717577b3e473ab57c4b9a0f4cb33/lms/templates/theme-variables.html#L166